runtime/jvmti/jvmtiClass.c

```diff
@@ -1784,9 +1785,20 @@ jvmtiGetConstantPool_translateCP(J9PortLibrary *privatePortLibrary, jvmtiGcp_tra
 			case J9CPTYPE_INSTANCE_METHOD:
 			case J9CPTYPE_STATIC_METHOD:
 			case J9CPTYPE_INTERFACE_METHOD:
-
+			case J9CPTYPE_INTERFACE_INSTANCE_METHOD:
+			case J9CPTYPE_INTERFACE_STATIC_METHOD:
```

Comment:
Can you define `U_8 cpType;` in this `case` block?  We can add `{` & `}` to possible to declare here.

 It keeps the scope of the variable small.